content/blog/how-to-set-up-credentials-to-authenticate-container-registry-in-hpe-greenlake-for-containers.md

```diff
@@ -0,0 +1,129 @@
+---
+title: How to Set up Credentials to Authenticate Docker Registry in HPE
+  GreenLake for Containers
+date: 2022-05-10T08:20:45.683Z
+author: Guoping Jia
+authorimage: /img/guoping.png
+tags:
+  - hpe-greenlake
+---
+## Introduction
+As one of HPE GreenLake Cloud Services, *HPE GreenLake for Containers* is an HPE-designed, implemented, owned, and operated private cloud that is built on HPE Ezmeral Container Platform, based on open source Kubernetes, and deployed at a customer site. In HPE GreenLake for Containers, it configures to use a gateway host, acting as a proxy sever, that carries client requests from deployed application service endpoints in the Kubernetes clusters. The gateway host maps the private IP endpoints of services running inside the Kubernetes clusters to external accessible IP addresses and ports. It provides better security by exposing only the gateway host IP address to the external while keeping all the others behind the firewall. However, when you create your application from Docker image in the Kubernetes cluster, your application pods may get stuck in the error state *`ErrImagePull`*. Below is a sample `ngnix` application deployment to the cluster and the received error message in the pod events:

```

Comment:
Replace HPE Ezmeral Container Platform with HPE Ezmeral Runtime Enterprise.
Also suggest to hotlink the HPE Ezmeral Runtime Enterprise --> link: https://www.hpe.com/us/en/software/ezmeral-runtime.html.

Also, hotlink HPE GreenLake for Containers. Link --> https://www.hpe.com/us/en/greenlake/containers.html

Reply:
Hi, Denis, 

Thank you for your review. Your comments have been addressed in the new patch.

Please let me know if there is still anything you want to change. 

Thank you !

Guoping